examples/models/models_orthographic_projection.c

```diff
@@ -0,0 +1,109 @@
+/*******************************************************************************************
+*
+*   raylib [models] example - Show the difference between perspective and orthographic projection 
+*
+*   This program is heavily based on the geometric objects example
+*
+*   This example has been created using raylib 1.0 (www.raylib.com)
+*   raylib is licensed under an unmodified zlib/libpng license (View raylib.h for details)
+*
+*   Copyright (c) 2014 Ramon Santamaria (@raysan5)
+*
+********************************************************************************************/
+
+#include "raylib.h"
+
+int main()
+{
+    // Initialization
+    //--------------------------------------------------------------------------------------
+    int screenWidth = 800;
+    int screenHeight = 450;
+
+    InitWindow(screenWidth, screenHeight, "raylib [models] example - geometric shapes");
+
+    const Camera perspective_camera = (Camera){{ 0.0f, 10.0f, 10.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 1.0f, 0.0f }, 45.0f, CAMERA_PERSPECTIVE };
```

Comment:
You can reduce code amount using just one Camera variable (without const) with perspective values and define values:
```
#define      FOVY_ORTHOGRAPHIC        10.0f
#define      FOVY_PERSPECTIVE         45.0f
```

So, you can then:

```
if(IsKeyPressed(KEY_SPACE)) 
{
     if(camera.type == CAMERA_PERSPECTIVE) 
     {
          camera.fovy = FOVY_ORTHOGRAPHIC;
          camera.type = CAMERA_ORTHOGRAPHIC;
     } 
     else 
     {
          camera.fovy = FOVY_PERSPECTIVE;
          camera.type = CAMERA_PERSPECTIVE;
     }
}
```